config/module.config.php

```diff
@@ -1,44 +1,85 @@
+use SlmQueueBeanstalkd\Factory\BeanstalkdWorkerControllerFactory;
+use SlmQueueBeanstalkd\Strategy\ClearObjectManagerStrategy;
+use SlmQueueBeanstalkd\Worker\BeanstalkdWorker;
 
-    'console'   => array(
-        'router' => array(
-            'routes' => array(
-                'slm-queue-beanstalked-worker' => array(
+return [
+    'service_manager' => [
+        'factories' => [
+            BeanstalkdWorker::class => WorkerFactory::class,
+            \Pheanstalk::class => \SlmQueueBeanstalkd\Factory\PheanstalkFactory::class
+        ]
+    ],
+    'controllers'     => [
+        'factories' => [
+            BeanstalkdWorkerController::class => BeanstalkdWorkerControllerFactory::class,
+            \Pheanstalk::class => \SlmQueueBeanstalkd\Factory\PheanstalkFactory::class
+        ],
+    ],
+    'console'         => [
+        'router' => [
+            'routes' => [
+                'slm-queue-Beanstalkd-worker'  => [
                     'type'    => 'Simple',
-                    'options' => array(
-                        'route'    => 'queue beanstalkd <queue> [--timeout=]',
-                        'defaults' => array(
-                            'controller' => 'SlmQueueBeanstalkd\Controller\BeanstalkdWorkerController',
+                    'options' => [
+                        'route'    => 'queue Beanstalkd <queue> [--timeout=] --start',
+                        'defaults' => [
+                            'controller' => BeanstalkdWorkerController::class,
                             'action'     => 'process'
-                        ),
-                    ),
-                ),
-            ),
-        ),
-    ),
-
-    'slm_queue' => array(
-        'beanstalkd' => array(
-            'connection' => array(
-                'host'    => '0.0.0.0',
-                'port'    => 11300,
-                'timeout' => 2
-            )
+                        ],
+                    ],
+                ],
+                'slm-queue-Beanstalkd-recover' => [
+                    'type'    => 'Simple',
+                    'options' => [
+                        'route'    => 'queue Beanstalkd <queue> --recover [--executionTime=]',
+                        'defaults' => [
+                            'controller' => BeanstalkdWorkerController::class,
+                            'action'     => 'recover'
+                        ],
+                    ],
+                ],
+                'slm-queue-Beanstalkd-stats' => [
+                    'type'    => 'Simple',
+                    'options' => [
+                        'route'    => 'queue Beanstalkd <queue> --stats',
+                        'defaults' => [
+                            'controller' => BeanstalkdWorkerController::class,
+                            'action'     => 'stats'
+                        ],
+                    ],
+                ],
+            ],
+        ],
+    ],
+    'slm_queue' => [
```

Comment:
I think everything below this line is some dummy / TODO code, that should now be removed. Right?